lib/auth/saml.go

```diff
@@ -166,6 +166,55 @@ func (a *Server) getSAMLConnectorAndProvider(ctx context.Context, req types.SAML
 	return connector, provider, nil
 }
 
+func (a *Server) resolveSAMLContext(ctx context.Context, samlResponse string) (types.SAMLAuthRequest, types.SAMLConnector, *saml2.SAMLServiceProvider, error) {
+	requestID, err := ParseSAMLInResponseTo(samlResponse)
+	if err != nil {
+		return types.SAMLAuthRequest{}, nil, nil, trace.Wrap(err)
+	}
+
+	if requestID == "" {
+		request := types.SAMLAuthRequest{
+			ID:               "idemeum-remote-launch",
```

Comment:
You can define UUID with all zeros as special IDEMEUM_REMOTE_LAUNCH and use it anywhere we have to compare and take decisions.